developers_affiliations3.txt

```diff
@@ -1934,8 +1934,10 @@ bmcmanus-apex: bmcmanus-apex!users.noreply.github.com
 	US Army until 2016-06-01
 	Independent from 2016-06-01 until 2019-06-01
 	Apex Fintech from 2019-06-01
-bmcstdio: bcustodio!mesosphere.io, bmcstdio!users.noreply.github.com
-	Mesosphere
+bmcustodio: bcustodio!mesosphere.io, bmcstdio!users.noreply.github.com, bmcustodio!users.noreply.github.com, brunomcustodio!gmail.com
+	Mesosphere until 2019-05-01
+	FORM3 from 2019-06-01 until 2019-04-01
```

Comment:
This looks like a bug you cannot make from=2016-06-01 and to 2019-04-01 (two months earlier) - I'll remove this affiliation in post merge.

Reply:
Sorry, that should have been 

Jun 2019 - Apr 2021 · 1 yr 11 mos